routes/web.php

```diff
@@ -13,14 +14,17 @@
 |
 */
 
-Route::get('/home', function () {
-    return view('welcome');
-});
 
-Route::get('/', function () {
-    return view('login');
-});
+Route::post('register',[AuthController::class,'store']);
+Route::view('/register','register');
+
+Route::view('/login','login');
```

Comment:
```suggestion
Route::view('/login','login')->name('login');
```
Always name routes and use name for clean codes.

Reply:
similar changes on all routes.